pyconpune/templates/index.html

```diff
@@ -72,19 +72,43 @@ <h3 class="text-white standard-margin-top">
                         <img class="speaker-image" src="{% static 'img/brettcannon.jpeg' %}"/>
                         <div class="speaker-name float-left blue-bg text-white full-width">Brett Cannon</div>
                         <div class="clearfix"></div>
-                        <p class="speaker-title float-left blue-bg text-white full-width">CPython Core Dev, Microsoft</p>
+                        <p class="speaker-title float-left blue-bg text-white full-width">CPython Core Dev</p>
```

Comment:
This line shouldn't be changed. Please check that you are not changing some code which was changed by some other commit.